staging/src/k8s.io/legacy-cloud-providers/aws/aws_loadbalancer.go

```diff
@@ -296,19 +295,65 @@ func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBa
 								Type:           aws.String("forward"),
 							}},
 						}
+
+						if mapping.SSLPolicy != "" {
+							modifyListenerInput.SslPolicy = aws.String(mapping.SSLPolicy)
+						}
+						elbCertificates := []*elbv2.Certificate{}
 						if mapping.FrontendProtocol == elbv2.ProtocolEnumTls {
 							if mapping.SSLPolicy != "" {
 								modifyListenerInput.SslPolicy = aws.String(mapping.SSLPolicy)
 							}
-							modifyListenerInput.Certificates = []*elbv2.Certificate{
-								{
-									CertificateArn: aws.String(mapping.SSLCertificateARN),
-								},
+							if mapping.SSLCertificateARN != "" {
+								for _, ARN := range strings.Split(mapping.SSLCertificateARN, ",") {
```

Comment:
let’s move for _, ARN := range strings.Split(mapping.SSLCertificateARN, ",") { into where we built the mappings
and change the datastructure to be certificateARNs or have certificateARNs and certificateARN at same time(if CLB’s code still relies on it and we don't plan to change it)
we should do the validations as soon as possible before the controller make changes
do the split during reconcile feels hacky to me.